drivers/source/DigitalIn.cpp

```diff
@@ -15,14 +15,17 @@
  * limitations under the License.
  */
 
-#include "drivers/AnalogIn.h"
+#include "drivers/DigitalIn.h"
```

Comment:
This file should be AnalogIn.cpp (and not DigitalIn.cpp).
The contents of both AnalogIn.cpp and  DigitalIn.cpp is correct but the history of the file will be odd.

Reply:
See below two (three commits each) logical attempts to fix the history. GitHub for some reason identifies `AnalogIn.cpp` as `DigitalIn.cpp` when `AnalogIn.cpp` is moved to `drivers/source`.
I am open to other suggestions how to fix the history.

Reply:
Weird git behaviour. Unless a git guru can suggest soemthing, I suggest that we leave as-is.